consensus/blacklist/blacklist_test.go

```diff
@@ -73,30 +72,3 @@ func TestBlacklistPeer(t *testing.T) {
 	require.True(t, hasWasCalled)
 	require.True(t, upsertWasCalled)
 }
-
```

Comment:
can you use export the start sweeping method in an export_test and keep this test?